group_vars/gitlab/production.yml

```diff
@@ -0,0 +1,5 @@
+---
+gitlab_pu_ldap_bind_dn: "{{ vault_gitlab_pu_ldap_bind_dn }}"
+gitlab_pu_ldap_password: "{{ vault_gitlab_ldap_password }}"
+gitlab_loadbalancer_domain_name: "pulgit.lib.princeton.edu"
```

Comment:
I would call this `gitlab.lib.princeton.edu`, I think - no reason to repeat the library part.